docs/cmd/tkn_pipeline_start.md

```diff
@@ -53,19 +53,21 @@ my-secret, my-empty-dir and my-volume-claim-template)
 ```
       --dry-run                       preview PipelineRun without running it
   -f, --filename string               local or remote file name containing a Pipeline definition to start a PipelineRun
+      --finally-timeout string        timeout for Finally TaskRuns
   -h, --help                          help for start
   -l, --labels strings                pass labels as label=value.
   -L, --last                          re-run the Pipeline using last PipelineRun values
   -o, --output string                 format of PipelineRun (yaml, json or name)
   -p, --param stringArray             pass the param as key=value for string type, or key=value1,value2,... for array type
+      --pipeline-timeout string       timeout for PipelineRun
       --pod-template string           local or remote file containing a PodTemplate definition
       --prefix-name string            specify a prefix for the PipelineRun name (must be lowercase alphanumeric characters)
   -r, --resource strings              pass the resource name and ref as name=ref
   -s, --serviceaccount string         pass the serviceaccount name
       --showlog                       show logs right after starting the Pipeline
       --skip-optional-workspace       skips the prompt for optional workspaces
       --task-serviceaccount strings   pass the service account corresponding to the task
-      --timeout string                timeout for PipelineRun
+      --tasks-timeout string          timeout for Pipeline TaskRuns
```

Comment:
IMO, this is a bit confusing for users to know which flags does what,

what about still using `--timeout` which will set both the finally/taskruns, which i think is what most users would care about. My gut feeling only a fringe of our users would care about setting a specific finally timeouts.

Reply:
@chmouel, we have deprecated this flag which means that the current flag still works but uses the deprecated timeout `pr.Spec.Timeout`. Whosoever is using this flag can still use this but will be getting a deprecation notice as
```
% ./tkn p start decision-pipeline --timeout 60m
Flag --timeout has been deprecated, please use --pipeline-timeout flag instead
? Value for param `input` of type `string`? taskB
PipelineRun started: decision-pipeline-run-6cf82

In order to track the PipelineRun progress run:
tkn pipelinerun logs decision-pipeline-run-6cf82 -f -n default
```

The flag `--pipeline-timeout` is the replacement of `--timeout` flag. This new flag uses the latest `spec.Timeouts.Pipeline` which is not backward compatible. Changing the behaviour of existing flag `--timeout` will break existing users who are using Pipelines version 0.36 or below

Reply:
what is the difference of --timeout as and --pipeline-timeout ?

Reply:
gotcha

/lgtm